### PR TITLE
Fixed Typos & Paths

### PR DESCRIPTION
**Pull request by Oedipsos**
- Fixed paths for case-sensitive systems
- Put exponents inside O(...) terms & cleared some parenthesis for consistency
- Corrected typos

### CODE REVIEW

File: notebooks/01_Introduction/01_02_TaylorExpansion.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ jupyter:
...
+**The first one** is related to the fact that most real numbers need an infinite numbers of digits after the decimal point to be properly represented. To store these numbers in the memory of a computer one, therefore, needs to cut their representation at some point beyond the decimal point. To quantify the numerical representation of data, one uses *precision*. Precision is usually measured *in bits or number of decimal digits*. For example, a number in *single precision* usually occupies 32 bits of computer memory. At the same time it is associated with 7 or fewer decimal digits. The number of digits kept is called the precision of the representation. To make a concrete example, in *single precision* and *double precision*, $\pi$ is approximated as follows:
```
Comment (YelyzavetaV) [inline notebooks/01_Introduction/01_02_TaylorExpansion.md:40]: I don't see, what has changed here, but it's marked as a change. Can you explain?

File: notebooks/01_Introduction/01_02_TaylorExpansion.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ In this course we always use double precision for real numbers, as this is the d
...
+In the context of the numerical solution of ordinary or partial diffential equations, **the second more significant approximation** comes from that the computer resources are limited. From the physical point of view, both time and space are continuous variables, but continuous data does not exist in the numerical world, as it would require infinite computer memory. Therefore, continuous variables are approximated using a discrete representation. For that purpose, we can for example introduce *grid points* at which the functions are evaluated. A small grid spacing results in a higher accuracy, but it comes at a higher computational cost. Some balance always has to be found.
```
Comment (YelyzavetaV) [inline notebooks/01_Introduction/01_02_TaylorExpansion.md:52]: I don't see, what has changed here, but it's marked as a change. Can you explain?

File: notebooks/01_Introduction/01_02_TaylorExpansion.md [hunk 3]
```diff
@@ -491,7 +491,7 @@ A nice feature of matplotlib is that it can display rendered LaTeX equations in
 
 ## Summary
 
-In this notebook we have introduced the concepts of numerical precision and numerical error. We also discussed different approaches to build numerical sequences in Python and learned about NumPy arrays - fundamental objects for scientific programming in Python. Using the Matplotlib package, we gave a trivial overview on 2D visualization and illustrated that the error term in Taylor series is of order $\mathcal{O}(\Delta x)^{n+1}$ when the last expansion term considered is of order $\mathcal{O}(\Delta x)^{n}$.
+In this notebook we have introduced the concepts of numerical precision and numerical error. We also discussed different approaches to build numerical sequences in Python and learned about NumPy arrays - fundamental objects for scientific programming in Python. Using the Matplotlib package, we gave a trivial overview on 2D visualization and illustrated that the error term in Taylor series is of order $\mathcal{O}(\Delta x^{n+1})$ when the last expansion term considered is of order $\mathcal{O}(\Delta x^{n})$.
```
Comment (YelyzavetaV) [inline notebooks/01_Introduction/01_02_TaylorExpansion.md:494]: I don't see, what has changed here, but it's marked as a change. Can you explain?

**Review by bknaepen — APPROVED** [review]
We will merge this pull request. After doing it, we will change the 'Style' directory to lowercase 'style' and make the corresponding changes in the files.

**Pull request merged**